collector/remote_fx.go

```diff
@@ -0,0 +1,362 @@
+// https://wutils.com/wmi/root/cimv2/win32_perfrawdata_counters_remotefxgraphics/
+
+type RemoteFxCollector struct {
+	// net
+	BaseTCPRTT               *prometheus.Desc
+	BaseUDPRTT               *prometheus.Desc
+	CurrentTCPBandwidth      *prometheus.Desc
+	CurrentTCPRTT            *prometheus.Desc
+	CurrentUDPBandwidth      *prometheus.Desc
+	CurrentUDPRTT            *prometheus.Desc
+	TotalReceivedBytes       *prometheus.Desc
+	TotalSentBytes           *prometheus.Desc
+	UDPPacketsReceivedPersec *prometheus.Desc
+	UDPPacketsSentPersec     *prometheus.Desc
+
+	//gfx
+	AverageEncodingTime                                *prometheus.Desc
+	FrameQuality                                       *prometheus.Desc
+	FramesSkippedPerSecondInsufficientClientResources  *prometheus.Desc
+	FramesSkippedPerSecondInsufficientNetworkResources *prometheus.Desc
+	FramesSkippedPerSecondInsufficientServerResources  *prometheus.Desc
+	GraphicsCompressionratio                           *prometheus.Desc
+	InputFramesPerSecond                               *prometheus.Desc
+	OutputFramesPerSecond                              *prometheus.Desc
+	SourceFramesPerSecond                              *prometheus.Desc
+}
+
+// NewRemoteFx ...
+func NewRemoteFx() (Collector, error) {
+	const subsystem = "remote_fx"
+	return &RemoteFxCollector{
+		// net
+		BaseTCPRTT: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "net_base_tcp_rtt"),
+			"Base TCP round-trip time (RTT) detected in seconds",
+			[]string{"session_name"},
+			nil,
+		),
+		BaseUDPRTT: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "net_base_udp_rtt"),
+			"Base UDP round-trip time (RTT) detected in seconds.",
+			[]string{"session_name"},
+			nil,
+		),
+		CurrentTCPBandwidth: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "net_current_tcp_bandwidth"),
+			"TCP Bandwidth detected in thousands of bits per second (1000 bps).",
+			[]string{"session_name"},
+			nil,
+		),
+		CurrentTCPRTT: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "net_current_tcp_rtt"),
+			"Average TCP round-trip time (RTT) detected in seconds.",
+			[]string{"session_name"},
+			nil,
+		),
+		CurrentUDPBandwidth: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "net_current_udp_bandwidth"),
+			"UDP Bandwidth detected in thousands of bits per second (1000 bps).",
+			[]string{"session_name"},
+			nil,
+		),
+		CurrentUDPRTT: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "net_current_udp_rtt"),
+			"Average UDP round-trip time (RTT) detected in seconds.",
+			[]string{"session_name"},
+			nil,
+		),
+		TotalReceivedBytes: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "net_total_received_bytes"),
+			"(TotalReceivedBytes)",
+			[]string{"session_name"},
+			nil,
+		),
+		TotalSentBytes: prometheus.NewDesc(
+			prometheus.BuildFQName(Namespace, subsystem, "net_total_sent_bytes"),
```

Comment:
Move `_total` here too
```suggestion
			prometheus.BuildFQName(Namespace, subsystem, "net_sent_bytes_total"),
```